pkgs/development/interpreters/bqn/cbqn/default.nix

```diff
@@ -54,8 +54,11 @@ stdenv.mkDerivation rec {
 
   buildFlags = [
     # interpreter binary
-    (lib.flatten (if enableSingeli then ["o3n-singeli" "f='-mavx2'"] else ["o3"]))
+    "o3"
+    "notui=1" # display build progress in a plain-text format
```

Comment:
Without this, what would happen?

Reply:
Without `notui=1`? Then the build progress uses more fancy progress output, rewriting the same line by outputting `\r` first. It's not strictly necessary here, but I guessed it might produce better logs in case there are problems in the build.